MachineLearningPrototype/dataCollection/collect_data.py

```diff
@@ -64,6 +64,12 @@ def store_issue_details(issue, repo, db):
 	store_quotation_result = store_on_fs(extract_quotation_content(issue['body']), issue_content_quotation_path)
 	if store_quotation_result:
 		issue_details['issue_content_quotation_path'] = issue_content_quotation_path
+
+	issue_quoted_jenkins_links =extract_jenkins_link_and_testname(issue['body'])[0]
+    	issue_quoted_test_names = extract_jenkins_link_and_testname(issue['body'])[1]
+    	if (issue_quoted_jenkins_links and issue_quoted_test_names) != []:
```

Comment:
We can check these two variables separately. Ignore whichever is empty, and save the non-empty one(s).